Remotion/ObjectBinding/Web/Res/HTML/BocAutoCompleteReferenceValue.jquery.js

```diff
@@ -354,13 +354,13 @@
             } else {
                 var isLastKeyPressBeforeBlurHandled = state.lastKeyPressCode == -1;
                 if (isLastKeyPressBeforeBlurHandled) {
-                    closeDropDownListAndSetValue($input.val());
+                    closeDropDownListAndSetValue($input.val(), false);

```

Comment:
💡We should introduce a {{focusInputAfterSelection}} varaible in line 355 to unify the state in lines 357 and 363.